Homework/Viktoriia Panina/lesson_34_sql/task1.sql

```diff
@@ -0,0 +1,50 @@
+select first_name, last_name, employees.department_id, department.department_name
+from employees
+join department
+on employees.department_id = department.department_id
```

Comment:
The task says: "...for each employee". This query does not return all employees.